services/authentication.py

```diff
@@ -200,4 +200,27 @@ def reset_password(session: Session, email: str, new_password: str, repeat_passw
         # print("test", passwordService.compare(old_password, test.password))
         return ResetPassword.SUCCESS
 
+    @staticmethod
+    def reset_password(session: Session, email: str, new_password: str, repeat_password: str):
```

Comment:
This would just allow me to enter *anyone's* email and change their password

Reply:
There is a reason we used email verification. Leave the old endpoint as it was and just make a new endpoint that is authenticated and updates the authenticated user's password

Reply:
The acceptance criteria does not specify functioning email. Leave the old flow as is